docs/Examples.md

```diff
@@ -76,34 +99,31 @@ metadata:
 spec:
   clusterMasterRef:
     name: cm
+  monitoringConsoleRef:
+    name: example_mc
 EOF
 ```
+This will automatically configure a cluster, with a predetermined number of index cluster peers generated automatically based upon the replication_factor (RF) set. This example includes the `monitoringConsoleRef` parameter used to define a monitoring console pod. 
```

Comment:
Can we improve this
`This example includes the `monitoringConsoleRef` parameter used to define a monitoring console pod.`
by saying
`This example assumes that a monitoring console pod "example_mc" has already been setup, and includes the monitoringConsoleRef parameter to reference to that monitoring console pod. For more details, see ### Monitoring Console`

Thoughts @eddikay ?

Reply:
Please review the most recent change. Thank you!